hack/make/.build-deb/docker-engine.install

```diff
@@ -9,3 +9,4 @@ contrib/init/systemd/docker.socket lib/systemd/system/
 contrib/mk* usr/share/docker-engine/contrib/
 contrib/nuke-graph-directory.sh usr/share/docker-engine/contrib/
 contrib/syntax/nano/Dockerfile.nanorc usr/share/nano/
+contrib/apparmor/* etc/apparmor.d/
```

Comment:
Right -- just installing the file isn't enough; you've got to have this
line (which is perfect), and a dh_apparmor line in "rules" (
http://manpages.debian.org/dh_apparmor; something like "dh_apparmor
--profile-name=bin.foo" probably in "override_dh_install").  On top of
that, we need to add "dh-apparmor" in "contrib/builder/deb/generate.sh".

https://codesearch.debian.net/results/dh_apparmor is a useful way to see
examples of how other packages invoke this same helper.